tendrl-alerting.spec

```diff
@@ -0,0 +1,74 @@
+Name: tendrl-alerting
+Version: 1.1
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Module for Tendrl Alerting
+Source0: %{name}-%{version}.tar.gz
+License: LGPLv2+
+URL: https://github.com/Tendrl/alerting
+
+BuildRequires: python-gevent
+BuildRequires: pytest
+BuildRequires: systemd
+BuildRequires: python-mock
+
+Requires: python-etcd
+Requires: python-dateutil
+Requires: gevent
+Requires: python-greenlet
+Requires: msgpack-python
+Requires: pytz
+Requires: flask
+Requires: Flask-API
+
+%description
+Python module for Tendrl alerting
+
+%prep
+%setup
+
+# Remove bundled egg-info
+rm -rf %{name}.egg-info
+
+%build
+%{__python} setup.py build
+
+# remove the sphinx-build leftovers
+rm -rf html/.{doctrees,buildinfo}
+
+%install
+%{__python} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+install -m  0755  --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/alerting
+install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/alerting
+install -m  0755  --directory $RPM_BUILD_ROOT%{_datadir}/tendrl/alerting
+install -Dm 0644 tendrl-alerting.service $RPM_BUILD_ROOT%{_unitdir}/tendrl-alerting.service
+install -Dm 0644 etc/tendrl/tendrl.conf.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/tendrl.conf
```

Comment:
Also the source file path should be etc/tendrl/alerting/alerting.conf.yaml.sample